{"name": "test_login_invalid_user[test_info0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert '' == '请选择机构!'\n  - 请选择机构!", "trace": "self = <test_cases.test_001_login.TestLogin object at 0x000002679FB17888>\ntest_info = {'expected': '请选择机构!', 'pwd': '123456', 'user': 'ceshiehdej'}\nget_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"94d19e4d15ded705e9d7375145464f76\")>\n\n    @pytest.mark.parametrize('test_info', login_data_invalid)\n    @allure.story(\"用户名不存在\")\n    def test_login_invalid_user(self, test_info, get_browser):\n        print(\"------执行登录用户名不存在测试用例------\")\n        driver = get_browser\n        expected = test_info['expected']\n        LoginPage(driver).login(test_info[\"user\"], test_info[\"pwd\"])\n        print(\"用户名：\", test_info[\"user\"])\n        print(\"密码：\", test_info[\"pwd\"])\n        invalid_msg = LoginPage(driver).get_invalid_msg()\n        print(\"实际结果\", invalid_msg)\n        print(\"预期结果\", expected)\n        try:\n            assert invalid_msg==expected\n        except AssertionError as e:\n>           raise e\n\ntest_001_login.py:61: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <test_cases.test_001_login.TestLogin object at 0x000002679FB17888>\ntest_info = {'expected': '请选择机构!', 'pwd': '123456', 'user': 'ceshiehdej'}\nget_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"94d19e4d15ded705e9d7375145464f76\")>\n\n    @pytest.mark.parametrize('test_info', login_data_invalid)\n    @allure.story(\"用户名不存在\")\n    def test_login_invalid_user(self, test_info, get_browser):\n        print(\"------执行登录用户名不存在测试用例------\")\n        driver = get_browser\n        expected = test_info['expected']\n        LoginPage(driver).login(test_info[\"user\"], test_info[\"pwd\"])\n        print(\"用户名：\", test_info[\"user\"])\n        print(\"密码：\", test_info[\"pwd\"])\n        invalid_msg = LoginPage(driver).get_invalid_msg()\n        print(\"实际结果\", invalid_msg)\n        print(\"预期结果\", expected)\n        try:\n>           assert invalid_msg==expected\nE           AssertionError: assert '' == '请选择机构!'\nE             - 请选择机构!\n\ntest_001_login.py:59: AssertionError"}, "parameters": [{"name": "test_info", "value": "{'user': 'ceshiehdej', 'pwd': '123456', 'expected': '请选择机构!'}"}], "start": 1600338471443, "stop": 1600338486531, "uuid": "ca77795e-7b44-41eb-988b-79baa685d116", "historyId": "075f4daf5ac51fdca2fb54fe12eff52f", "testCaseId": "cd0682f6a139d5392c369302963db253", "fullName": "test_cases.test_001_login.TestLogin#test_login_invalid_user", "labels": [{"name": "story", "value": "用户名不存在"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_001_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-G986QGRS"}, {"name": "thread", "value": "13864-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_001_login"}]}